src/utils.js

```diff
@@ -116,6 +116,10 @@ function getCardColors({
   return { titleColor, iconColor, textColor, bgColor };
 }
 
+const fn = () => {};
+// return console instance based on the environment
+const logger = process.env.NODE_ENV !== "test" ? console : { log: fn, error: fn };
```

Comment:
Not sure about the logger function but i actually do use console.logs for debugging some stuff on the preview

Reply:
It will work as console work. It's just a console instance but it won't print anything in test environment.

Reply:
Oh it's !== sorry missed that